README.md

```diff
@@ -110,8 +122,8 @@ migration process.
 
 In practice this means that ZDM Proxy supports the following cluster versions (as Origin and / or Target):
 
-- Apache Cassandra from 2.1+ up to (and including) Apache Cassandra 4.x. Apache Cassandra 2.0 support will be introduced 
-when protocol version v2 is supported.
+- Apache Cassandra from 2.1+ up to (and including) Apache Cassandra 4.x.
+- Apache Cassandra 2.0 up to 2.1.
```

Comment:
Just edit the previous line to say `2.0+ up...` instead of `2.1+` and delete this one